KarhooUISDK/Screens/CheckoutScreen/Components/Loyalty/LoyaltyContainerWithBallance.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  LoyaltyContainerWithBalance.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 26/01/2023.
+//  Copyright © 2023 Flit Technologies Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct LoyaltyContainerWithBalance<Content: View>: View {
```

Comment:
@BartlomiejSopalaKarhoo do you think we could hide those state-views behind some sort of namespace, so it's inaccessible in global scope? This way we'd be more confident the view instance will not be used directly, which, is most of the cases, is an improper way to implementation usage.